us.zoom.Zoom.json

```diff
@@ -11,9 +11,11 @@
         "--socket=pulseaudio",
         "--share=network",
         "--device=all",
-        "--filesystem=home",
         "--env=QT_QPA_PLATFORM=",
-        "--own-name=org.kde.*"
+        "--own-name=org.kde.*",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-videos",
+        "--persist=.zoom"
```

Comment:
Zoom also dumps settings in `.config/zoom` and `.config/Unknown Organization`.

Reply:
That should be automatically be taken care of by flatpak through xdg-config-home preservation. You normally only need special permissions when stuff is writing to non-xdg locations or when you want holes in sandbox.

Reply:
Uh, thanks for the clarification, didn't know that, but it makes sense.